exercises/connect/ConnectTest.cs

```diff
@@ -1,124 +1,86 @@
-﻿using Xunit;
-using System.Linq;
+// This file was auto-generated based on version 1.0.0 of the canonical data.
+
+using Xunit;
 
 public class ConnectTest
 {
-    private static string MakeBoard(string[] board)
+    [Fact]
+    public void An_empty_board_has_no_winner()
     {
-        return string.Join("\n", board.Select(x => x.Replace(" ", "")));
+        var board = new[] { ".....", ".....", ".....", ".....", "....." };
```

Comment:
For the Connect case, it is quite important that the input strings are aligned nicely under each other, to allow users to see the pattern that needs to be connected. By default, the generator framework does not support this. but perhaps you can "borrow" from [this generator](https://github.com/exercism/csharp/pull/457/files#diff-2de73a257c3f6aa9810bdd1be46c7ab4R37)?